backend/FwLite/MiniLcm.Tests/AutoFakerHelpers/ObjectWithIdOverride.cs

```diff
@@ -0,0 +1,22 @@
+﻿using MiniLcm.Models;
+using Soenneker.Utils.AutoBogus.Context;
+using Soenneker.Utils.AutoBogus.Generators;
+
+namespace MiniLcm.Tests.AutoFakerHelpers;
+
+public class ObjectWithIdOverride : AutoFakerGeneratorOverride
+{
+    public override bool CanOverride(AutoFakerContext context)
+    {
+        return context.GenerateType.IsAssignableTo(typeof(IObjectWithId));
+    }
+
+    public override void Generate(AutoFakerOverrideContext context)
+    {
+        if (context.Instance is IObjectWithId obj)
+        {
+            obj.DeletedAt = null;
```

Comment:
if someone calls update with a deleted object what should we do? throw? delete it?

Reply:
I say we throw.

If the object wasn't/isn't already deleted, it would mean that they came up with a deleted date themselves, which could be anything. That seems really weird.

If they know that it was deleted and want to make changes to it...that's also weird.